aspnet/fundamentals/middleware.rst

```diff
@@ -2,24 +2,24 @@
 
 Middleware
 ==========
-By `Steve Smith`_
+By `Steve Smith`_ and `Rick Anderson`_
 
-Small application components that can be incorporated into an HTTP request pipeline are known collectively as middleware. ASP.NET 5 has integrated support for middleware, which are wired up in an application's ``Configure`` method during :doc:`startup`.
+Middleware is defined in the `OWIN specification <http://owin.org/html/spec/owin-1.0.1.html>`__ as pass through components that form a pipeline between a server and application to inspect, route, or modify request and response messages for a specific purpose. Middleware consists of application components are incorporated into the ASP.NET HTTP pipeline. 
```

Comment:
> Middleware is defined in the `` `OWIN specification <http://owin.org/html/spec/owin-1.0.1.html>`__  ``

We should remove this reference to OWIN. ASP.NET Core middleware and OWIN middleware are *not* the same